torchgeo/models/resnet.py

```diff
@@ -52,6 +52,13 @@
     data_keys=["image"],
 )
 
+# https://github.com/microsoft/torchgeo/blob/8b53304d42c269f9001cb4e861a126dc4b462606/torchgeo/datamodules/ssl4eo_benchmark.py#L43 # noqa: E501
+_ssl4eo_l_transforms = AugmentationSequential(
+    K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)),
+    K.RandomResizedCrop((224, 224), scale=(0.6, 1.0)),
+    data_keys=["image"],
```

Comment:
I'm never sure if we're supposed to append all possible data keys here or only the ones we're actually using. Like if we use this with a mask, we want the mask to be cropped too.

Reply:
Are we defining these transforms just as an "information" guideline or are they somewhere automatically integrated and used? I wouldn't think so since the torchgeo datamodules defined the augmentations. So if people have to write their own datamodule anyway then they will have to adapt the augmentations for their use case as well. Or how is this `Weights` attribute supposed to be used?

Reply:
AFAIK, they aren't automatically used by anything, but users can programmatically use them in their code. The idea is that these are reference transforms that can be used with the weights, but users are free to adapt them. Note that not all users will be using Lightning, some may directly import the weights and use the transforms in their existing pipelines.